File: app/src/main/java/com/a60n1/ejashkojme/models/Friends.java
package com.a60n1.ejashkojme.models;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class Friends {

    public String date;

    public Friends() {

    }

    public Friends(String date) {
        this.date = date;
    }
}

File: app/src/main/java/com/a60n1/ejashkojme/models/Friends.kt
package com.a60n1.ejashkojme.models

import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
class Friends {
    var date: String? = null

    @Suppress("unused")
    constructor()

    @Suppress("unused")
    constructor(date: String?) {
        this.date = date
    }
}